### PR TITLE
fix(subgraph): improve `deploy-all` task

### PR DESCRIPTION
**Pull request by clemsos**
# Description

Fix task to deploy all subgraphs

```
yarn deploy-all
```

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by clemsos** [comment]
ok that worked :) all v2 subgraphs deployed on thegraph hosted version

**Comment by julien51** [comment]
> ok that worked :) all v2 subgraphs deployed on thegraph hosted version

Wow! That's very cool! Links maybe?

File: subgraph/bin/networks.js [hunk 1]
```diff
@@ -6,8 +6,11 @@ const networksConfig = require('@unlock-protocol/networks')
 
 const networkFilePath = path.join(__dirname, '..', 'networks.json')
 
+const networkName = (n) =>
+  n === 'polygon' ? 'matic' : n === 'arbitrum' ? 'arbitrum-one' : n
```
Comment (julien51) [inline subgraph/bin/networks.js:10]: That's annoying. Do you think we could update the network config rather by changing the `subgraph` property to an object that has `endpoint` and `networkName` ?

File: subgraph/bin/thegraph [hunk 2]
```diff
@@ -17,51 +18,58 @@ const executeCommand = async (command) => {
...
-
   if (!network) {
     throw new Error('Missing network name')
   }
 
   const graphNode = 'https://api.thegraph.com/deploy/'
   const ipfs = 'https://api.thegraph.com/ipfs/'
-  const subgraphName = `unlock-protocol/${network}-v2`
+
+  const subgraphName = `unlock-protocol/${
```
Comment (julien51) [inline subgraph/bin/thegraph:58]: same here? We would add `subgraphName` ?

**Review by julien51 — APPROVED** [review]

File: subgraph/bin/thegraph (hunk 2 above)
Comment (clemsos) [reply subgraph/bin/thegraph:58]: that can actually be parsed from the endpoint

File: subgraph/bin/networks.js (hunk 1 above)
Comment (clemsos) [reply subgraph/bin/networks.js:10]: yes possible. It will just require to change accross the board in locksmith etc. Also we may end a `endpointV2` then

**Comment by clemsos** [comment]
> Wow! That's very cool! Links maybe?

so far the links are the same as v1 with the `-v2` suffix added. i.e. `https://api.thegraph.com/subgraphs/name/unlock-protocol/goerli-v2`

File: subgraph/bin/networks.js (hunk 1 above)
Comment (julien51) [reply subgraph/bin/networks.js:10]: yeah I like that

Comment (clemsos) [reply subgraph/bin/networks.js:10]: see https://github.com/unlock-protocol/unlock/pull/9709

**Pull request merged**